regex-assembly/932175.ra

```diff
@@ -14,10 +14,12 @@
 
   ##! match white space between command and arguments
   \b\s+
+  \b\s+[+\-][a-z]+\s+
+  \b\s+[+\-][a-z]+\+\s+
   ##!=>
 
   ##! match the alias name
-  ['"\w!%,@]+
+  [.'"\w!%\-,@]+
```

Comment:
It would be great to have a comment here that explains the pattern. Why do you need `.` and `-` here?

Reply:
To match these: https://github.com/coreruleset/coreruleset/pull/3740#issuecomment-2184004296